### DISCUSSION TITLE
[MacOS] inotify instance limit reached

### DISCUSSION THREAD

**Discussion by ghost** [containers/podman#19847] [Q&A] [answered]
This happens a lot on my dev instances especially using `kind` clusters.

It's pretty easy to solve it doing `podman machine ssh` and then `echo fs.inotify.max_user_instances=524288 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p`

There's a way to force this setup in vm by podman itself during `init`?
If not it's worth opening a bug to increase this value by default?

**Comment by vrothberg** [comment, marked as answer]
Thanks for reaching out, @valeriano-manassero!  Yes, please open an issue for it.
